pkg/server/controllers.go

```diff
@@ -448,6 +450,9 @@ func (s *Server) installApiResourceController(ctx context.Context, config *rest.
 			metav1.GroupResource{Group: workload.GroupName, Resource: "workloadclusters"},
 			metav1.GroupResource{Group: apiresourceapi.GroupName, Resource: "apiresourceimports"},
 			metav1.GroupResource{Group: apiresourceapi.GroupName, Resource: "negotiatedapiresources"},
+			metav1.GroupResource{Group: apis.GroupName, Resource: "apiresourceschemas"},
+			metav1.GroupResource{Group: apis.GroupName, Resource: "apiexports"},
+			metav1.GroupResource{Group: apis.GroupName, Resource: "apibindings"},
```

Comment:
are these really needed for the api-resource controllers?

Reply:
No, I should be able to undo this (was dealing with e2e issues)

Reply:
done